### DISCUSSION TITLE
关于解锁码解绑问题

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#4343] [提问] [answered]
如题，换计算机了，在爱发电平台上已经绑定的识别码怎么解绑?

**Comment by WForst-Breeze** [comment, marked as answer]
需要更新时，直接使用 更新+新识别码 即可换绑

**Comment by Ad-closeNN** [comment]
?需要绑定吗
每次更新只需 `更新 识别码`

**Reply by MoYuan-CN** [reply]
实际上在你 `更新 识别码` 的时候伯特会记住你的

**Comment by Ad-closeNN** [comment]
更新那么多次现在才知道w(ﾟДﾟ)w





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Hex-Dragon/PCL2"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月25日(星期四) 下午3:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Hex-Dragon/PCL2] 关于解锁码解绑问题 (Discussion #4343)





 
实际上在你 更新 识别码 的时候伯特会记住你的
